src/py42/settings/__init__.py

```diff
@@ -9,6 +9,7 @@
 verify_ssl_certs = True
 
 items_per_page = 500
+security_events_per_page = 10000
```

Comment:
I considered naming this `security_items_per_page`, but I think `security_events_per_page` better indicates it's intended use.